### PR TITLE
Update azure-ai-for-developers-dotnet.md

### PR DESCRIPTION
**Pull request by MilanKaur-01**
Proposing a change to include an end-to-end sample built by Azure Communication Services team that uses Azure AI and RAG

### CODE REVIEW

**Comment by Court72** [comment]
@mcleanbyron

Can you review the proposed changes? Note that the validation warning must be resolved before merging.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/intro/includes/azure-ai-for-developers-dotnet.md [hunk 1]
```diff
@@ -6,9 +6,12 @@ Azure AI reference templates provide you with well-maintained, easy to deploy re
...
+|[Get started with the .NET enterprise chat sample using RAG](/dotnet/azure/ai/get-started-app-chat-template)|An article that walks you through deploying and using the [Enterprise chat app sample for .NET](https://github.com/Azure-Samples/azure-search-openai-demo-csharp). This sample is a complete end-to-end solution demonstrating the [Retrieval-Augmented Generation (RAG) pattern](/azure/search/retrieval-augmented-generation-overview) running in Azure, using Azure AI Search for retrieval and Azure OpenAI large language models to power ChatGPT-style and Q&A experiences. [Demo video](https://aka.ms/azai/net/video) |
+|[Build GPT-automated customer support](https://aka.ms/acsaisampleapp)| This sample uses [Azure Communication Services](aka.ms/acsdocs) along with RAG, Azure AI Services and Azure Open AI to build a GPT powered customer support system that uses chat, calling and Email. [Demo video](https://aka.ms/acs-mech) |
```
Comment (alexwolfmsft) [inline articles/intro/includes/azure-ai-for-developers-dotnet.md:10]:
```suggestion
|[Build GPT-automated customer support](https://aka.ms/acsaisampleapp)| This sample uses [Azure Communication Services](https://aka.ms/acsdocs) along with RAG, Azure AI Services and Azure Open AI to build a GPT powered customer support system that uses chat, calling and Email. [Demo video](https://aka.ms/acs-mech) |
```

This is causing a build warning

**Comment by KarlErickson** [comment]
@alexwolfmsft is this ready to merge? Thanks!

**Comment by MilanKaur-01** [comment]
Needs more work - discussed with AI content team

**Pull request closed without merging**